css/home.css

```diff
@@ -0,0 +1,87 @@
+html,
+body {
+  height: 83.2%;
+  font-family: "Actor", sans-serif;
```

Comment:
the height property is useless, try to stop it, nothing will happen

Reply:
Dear Sara, 
The idea is to make the height of the page = device veiw height
I do not need any scroll down except for the div box

When I remove it, the page will expand to number of tasks

Reply:
Doaa Also, Added that using this will solve the problem better than height as %
overflow: hidden